src/js/components/FilterCriteria.jsx

```diff
@@ -109,20 +128,44 @@ class FilterCriteria extends Component {
     ev.preventDefault()
     dispatch(removeFilter(index))
   }
-
+  
+  handleTouchTap () {
+    this.setState({
+      expanded: !this.state.expanded
+    })
+  }
+  
   render () {
     const {
       fields,
       filters,
+      headerStyle,
+      headerText,
+      onClickFilter,
       style,
       wrapperStyle
     } = this.props
 
     return (
       <div style={wrapperStyle}>
-        <h3>Select filter criteria (optional)</h3>
-        {
-          filters.map((filter, i) => (
+        <h3 style={headerStyle}>
+          {headerText}
+          <CardExpandable
```

Comment:
It seems that the entire `CardExpandable` component shouldn't exist if `onClickFilter` isn't provided, not just the button.